### PR TITLE
Add a more extensive logging module and turn it on by default

### PR DESCRIPTION
**Pull request by paetling**
review @jsingle

### CODE REVIEW

File: lib/logging.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+var winston = require("winston");
+var config = require("./config")
+
+var consoleLogger = new winston.transports.Console({"timestamp": true});
+var fileLogger = new winston.transports.File({"filename": config.log_file,
+    "timestamp": true
+});
+
+var logger = new winston.Logger({
+    "emitErrs": false,
```
Comment (jsingle) [inline lib/logging.js:10]: what's this field?

File: middleware/logging.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+var expressWinston = require("express-winston");
+var logging = require("../lib/logging");
+var config = require("../lib/config")
+
+var winstonMiddleware = expressWinston.logger({
+    "transports": [logging.consoleLogger, logging.fileLogger],
```
Comment (jsingle) [inline middleware/logging.js:6]: 👍 logging to both

File: lib/logging.js (hunk 1 above)
Comment (paetling) [inline lib/logging.js:10]: This says what should happen if the logger fails to log to the described location. At the moment I am just supressing the errors

**Comment by jsingle** [comment]
Yay

**Pull request merged**